source/layouts/api_layout.html.erb

```diff
@@ -10,7 +10,10 @@
         <% end %>
       </ul>
     </li>
-    <li><%= sidebar_link 'Search API', '/apis/search-api.html' %></li>
+    <li><%= sidebar_link 'Search API', '/apis/search/search-api.html' %></li>
+    <ul>
```

Comment:
This one is supposed to be top level. I'm trying to follow the same pattern as publishing api - some kind of overview page with a bunch of child pages.